### DISCUSSION TITLE
请问有什么方法可以把DatePicker弹窗组件，点击打开时，右侧的结束日期选择，设置为当前月

### DISCUSSION THREAD

**Discussion by CatchLF** [ant-design/ant-design#40904] [Q&A]
<img width="703" alt="image" src="https://user-images.githubusercontent.com/114169497/221125067-bcf9daa6-abc7-4240-8f3c-a390cefefefc.png">
请问不修源代码的时候做到打开datePicker控件时，右侧可选择日期为当前月份（因为该功能做的主要是历史记录查询，未来日一般时不需要进行选择的）

**Comment by Wxh16144** [comment]
> 右侧可选择日期为当前月份

意思是左右两侧日期面板都是同一个月份么？

**Reply by CatchLF** [reply]
不是的，是想做到选中日期以后，弹窗的效果和图一一样
![00E3EDFB-D7B5-4C25-83AE-DDFA8569984C_1_102_o](https://user-images.githubusercontent.com/114169497/221149507-1a4a1b10-16a0-4358-a819-7697a6e42e9b.jpeg)
图一⬆️
![B0AE1317-C05F-4047-9B37-95F314644FC1_1_102_o](https://user-images.githubusercontent.com/114169497/221150247-15ed49c3-f196-40ba-8b66-cdb81903b2ef.jpeg)

现在的效果⬆️

**Reply by CatchLF** [reply]
使用defaultPickerValue可以设置不选择日期时日期的范围，但选择日期以后，defaultPickerValue就会失效。
